main.go

```diff
@@ -224,11 +227,13 @@ func isNewLogLine(line string) bool {
 
 var (
 	loggerSourceType string
+	tcpOutUrl        string
 	displayVersion   bool
 
 	hostname string = ""
 
-	version string = "0.0.6"
+	version   string = "0.0.6"
+	tcpLogger *TCPLogger
```

Comment:
what is the purpose of initializing this variable instead of just using the struct that `NewTCPLogger` returns? is it for readability?

Reply:
Ah wait. I see that you have it so it can be global variable, and be able to be passed in as an arg for `HandlePostgresLogLine`.